AsyncDisplayKit/ASCellNode.mm

```diff
@@ -298,8 +300,16 @@ - (void)handleVisibilityChange:(BOOL)isVisible
   
   ASDisplayNode *owningNode = scrollView.asyncdisplaykit_node;
   if ([owningNode isKindOfClass:[ASCollectionNode class]]) {
+    NSIndexPath *ip = [(ASCollectionNode *)owningNode indexPathForNode:self];
+    if (ip != nil) {
+      [result addObject:@{ @"indexPath" : ip }];
+    }
```

Comment:
This is sort of unrelated and should have been here before. Really helps logging.